skimage/transform/_geometric.py

```diff
@@ -293,23 +282,10 @@ def __call__(self, coords):
         coords_homogeneous = np.column_stack([coords, np.ones(coords.shape[0])])
         return coords_homogeneous @ self.params.T
 
-    def inverse(self, coords):
-        """Apply inverse transformation.
-
-        Parameters
-        ----------
-        coords : (N, 2) array_like
-            Destination coordinates.
-
-        Returns
-        -------
-        coords : (N, 3) array
-            Epipolar lines in the source image.
-
-        """
-        coords = np.asarray(coords)
-        coords_homogeneous = np.column_stack([coords, np.ones(coords.shape[0])])
-        return coords_homogeneous @ self.params
+    @property
+    def inverse(self):
+        """Return a transform object representing the inverse."""
+        return type(self)(matrix=self.params.T)
```

Comment:
I'm not so familiar with epipolar geometry, but intuitively I would understand the term "inverse" on all transform classes as something capable of the backward transform from DST to SRC. I see this independent from whether the parameters are polynomial coefficients or a fundamental matrix; and whether the backwards transformation is actually performed by a transposed matrix instead of an inverse matrix.

Other opinions would be helpful!

Reply:
Reading your response and thinking about it again, inverse *is* a pretty accepted term describing the opposite operation of a transformation (e.g. `ifft`, `iradon`). Let's keep it like this.